AddressBook_/AddressBook_/DetailViewController.swift

```diff
@@ -0,0 +1,368 @@
+    
+    let backButton          = UIButton()
+    let saveButton          = UIButton()
+    
+    let memberImageView     = UIImageView()
+    let addImageButton      = UIButton()
+    
+    let nameLabel           = UILabel()
+    let nameTextField       = UITextField()
+    let phoneLabel          = UILabel()
+    let phoneTextField      = UITextField()
+    
+    let favoriteLabel       = UILabel()
+    let yesButton           = UIButton()
+    let noButton            = UIButton()
+    
+    let memoLabel           = UILabel()
+    let memoTextView        = UITextView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        addAction()
+        initialSetup()
+        makeUI()
+    }
+    
+    // ********************************
+    // MARK: - addAction
+    // ********************************
+    func addAction() {
+        backButton.addTarget(self, action: #selector(backButtonClicked), for: .touchUpInside)
+        saveButton.addTarget(self, action: #selector(saveButtonClicked), for: .touchUpInside)
+        addImageButton.addTarget(self, action: #selector(addImageButtonClicked), for: .touchUpInside)
+        yesButton.addTarget(self, action: #selector(yesButtonClicked), for: .touchUpInside)
+        noButton.addTarget(self, action: #selector(noButtonClicked), for: .touchUpInside)
+    }
+    
+    // ********************************
+    // MARK: - backButtonClicekd
+    // ********************************
+    @objc func backButtonClicked() {
+        self.dismiss(animated: true)
+    }
+
+    // ********************************
+    // MARK: - saveButtonClicked
+    // ********************************
+    @objc func saveButtonClicked() {
+        // 미입력시 alert창 띄우기
+        if nameTextField.text == "" {
+            showAlert(title: "미입력", message: "이름을 입력해주세요")
+            return
+        } else if phoneTextField.text == "" {
+            showAlert(title: "미입력", message: "비밀번호를 입력해주세요")
+            return
+        } else if yesButton.isSelected == false && noButton.isSelected == false {
+            showAlert(title: "미입력", message: "즐겨찾기 여부를 선택해주세요")
+        }
+        
+        // 새로운 멤버를 생성해서 저장
+        guard let image = memberImageView.image else { return }
+        guard let name  = nameTextField.text    else { return }
+        guard let phone = phoneTextField.text   else { return }
+        guard let memo  = memoTextView.text     else { return }
+        var isFavorite  = false
+
+        if yesButton.isSelected { isFavorite = true }
+        
+        MemberManger.shared.append(image: image, name: name, phoneNumber: phone, memo: memo, isFavorite: isFavorite)
+        
+        showAlert(title: "저장", message: "저장이 완료되었습니다", dismissPresentingView: true)
+        delegate?.reloadTable()
+    }
+    
+    // ********************************
+    // MARK: - addImageButtonClicked
+    // ********************************
+    @objc func addImageButtonClicked() {
+        
```

Comment:
비워둔 함수에는 항상 향후 작성할 내용에 대해 코멘트를 남겨주세요!
```suggestion
        // TODO: 이미지 추가 기능 구현 예정
```

Reply:
코멘트가 없는 경우는 
- 함수를 잘못 작성하였는지
- 사용하지 않은 함수인지
- 미래에 구현 예정이라 비워둔것인지
- 현재 구현하다 막힌 것인지
등을 파악할 수 없습니다.
현재 어떤 상태인지 명시해주시면 됩니다.